src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -21,22 +34,7 @@ export default {
       },
     },
     label: {
-      control: {
-        options: ['headline', 'claims', 'Test'],
-        type: 'string',
-      },
-    },
-    size: {
-      control: {
-        options: ['small', 'medium', 'large'],
-        type: 'radio',
-      },
-    },
-    variant: {
-      control: {
-        options: ['outlined', 'standard', 'filled'],
-        type: 'radio',
-      },
```

Comment:
Again probably no need to adjust this, we can just pick one (outlined) and keep the design consistent.